### PR TITLE
Update README.md about installing / adjusting firewall on Fedora,Centos and RHEL

### PR DESCRIPTION
**Pull request by itamarjp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,14 @@ apt-get install xrdp
 
 ### RedHat / CentOS / Fedora
 ```bash
-yum install xrdp
+yum -y install xrdp
+dnf -y install xrdp
```
Comment (proski) [inline README.md]: Please add a note that yum is for older distros.

File: README.md [hunk 2]
```diff
@@ -47,7 +47,14 @@ apt-get install xrdp
...
+
+After installing Xrdp in Centos, RHEL or Fedora  you need to adjust the firewall
```
Comment (proski) [inline README.md]: This should not be in the code block.

File: README.md [hunk 3]
```diff
@@ -47,7 +47,14 @@ apt-get install xrdp
...
 ### RedHat / CentOS / Fedora
 ```bash
-yum install xrdp
+yum -y install xrdp
+dnf -y install xrdp
+
+After installing Xrdp in Centos, RHEL or Fedora  you need to adjust the firewall 
+
+# firewall-cmd --permanent --zone=public --add-port=3389/tcp
+# firewall-cmd --reload
```
Comment (proski) [inline README.md]: Why are those commands commented out?

File: README.md (hunk 1 above)
Comment (metalefty) [inline README.md]: Why you're adding `-y` ?

Comment (itamarjp) [inline README.md]: to install without asking to confirm

**Comment by itamarjp** [comment]
I will merge please update on top of it if needed.

**Comment by proski** [comment]
I believe it is the responsibility of the PR author to address all objections. In some cases it may be OK to push changes and promise fixes later. It's much worse to merge the PR as is and ask *others* to submit changes. That shifts the responsibility to other developers, who may have completely different priorities and perspectives. I, for one, would prefer to have that information in the Wiki.

**Comment by proski** [comment]
The requested fixes are part of #654

File: README.md (hunk 1 above)
Comment (metalefty) [inline README.md]: I know what `-y` does but why it's added?
I think users should confirm it on their own responsibility.

`apt` also ask users to confirm installation. Adding it only to `yum/dnf` is inconsistent.

**Pull request merged**